ci/pod/e2e-sse.yaml

```diff
@@ -28,13 +28,13 @@ spec:
     image: milvusdb/knowhere-cpu-build:amd64-ubuntu22.04-20240914-814ecaa4
     tty: true
     args: ["cat"]
-    # resources:
-    #   requests:
-    #     memory: "16Gi"
-    #     cpu: "4"
-    #   limits:
-    #     memory: "16Gi"
-    #     cpu: "4"
+    resources:
+      requests:
+        memory: "3Gi"
```

Comment:
Per our previous talk, the host is 2C4G. Why we made it 16G before?

Reply:
the host is 8c4g, Huangliang set no resource limit before